### ISSUE TITLE
Stripe integration

### ISSUE THREAD

**Issue by ixahmedxi** [ixahmedxi/orbitkit#7]
We should create a `packages/stripe` package that integrates stripe functionality and let's the web application consume that to set up payment processing capabilities for the web application

**Comment by tconroy** [comment]
Howdy! Just started poking around Orbitkit and what I see I really like, but this feels like the last critical missing piece IMO - Would love to see this added!

Do you have any opinions on how the `stripe` package can/should be structured? I could look into opening a PR to assist with this when I get some free time.

**Comment by ixahmedxi** [comment]
I agree as well that this does feel like the last missing critical piece, I have been looking at referencing of doing this the proper way and I think it would be:

- `packages/stripe` that handles all of the stripe stuff.
- consumed by the `api` package and inside the `web` app.
